Refit/Buffers/PooledBufferWriter.Stream.NETStandard21.cs

```diff
@@ -0,0 +1,73 @@
+﻿#if NETSTANDARD2_1
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+#nullable enable
```

Comment:
Nit: We don't need this as the whole codebase has nullable enabled.

Reply:
That's weird, I'm not seeing the `<Nullable>enable</Nullable>` tag in the `.csproj` file, and in fact Visual Studio is showing warnings over the fields marked as nullable if I remove that directive 🤔

I might very well be missing something obvious here. Should I ignore the warnings and remove that directive anyway?

Reply:
Hmm, you're right. I thought it was already there. Can you please add `<Nullable>enable</Nullable>` to https://github.com/reactiveui/refit/blob/master/Directory.build.props instead?

Also, there's a merge conflict that needs to be resolved as well.

Reply:
@clairernovotny 
Are you sure of that? Currently it leads to tons of `Dereference of a possibly null reference` across all solution.
If we think that nullability should be enabled, I'd suggest to do that in separate PR(s), will all warnings resolved.

Reply:
I saw this last comment only after pushing those changes in  https://github.com/reactiveui/refit/pull/836/commits/72f1d4419623ec3b9c870a514a98f565eb00e92e 😅

Of course though, if you change your mind I can revert it.

Reply:
You're right. Best to do that as a separate PR.

Reply:
Alright, reverted those changes in https://github.com/reactiveui/refit/pull/836/commits/c04b0f27d97fcc5516187d2f187ae66c20151659 👍